plugins/export-workbook/src/plugin.ts

```diff
@@ -8,36 +8,23 @@ import * as fs from 'fs'
 import path from 'path'
 import * as R from 'remeda'
 import * as XLSX from 'xlsx'
-import { sanitize, sanitizeExcelSheetName } from './utils'
+import {
+  createXLSXSheetOptions,
+  sanitize,
+  sanitizeExcelSheetName,
+} from './utils'
+import { PluginOptions } from './options'
 
 const api = new FlatfileClient()
 
-/**
- * Plugin config options.
- *
- * @property {string} jobName - name of the job
- * @property {string[]} excludedSheets - list of sheet names to exclude from the exported data.
- * @property {string[]} excludeFields - list of field names to exclude from the exported data. This applies to all sheets.
- * @property {Flatfile.Filter} recordFilter - filter to apply to the records before exporting.
- * @property {boolean} includeRecordIds - include record ids in the exported data.
- * @property {boolean} autoDownload - auto download the file after exporting
- * @property {boolean} debug - show helpul messages useful for debugging (use intended for development).
- */
-export interface PluginOptions {
-  readonly jobName?: string
-  readonly excludedSheets?: string[]
-  readonly excludeFields?: string[]
-  readonly recordFilter?: Flatfile.Filter
-  readonly includeRecordIds?: boolean
-  readonly autoDownload?: boolean
-  readonly debug?: boolean
-}
+const LOG_NAME = '@flatfile/plugin-export-workbook'
```

Comment:
I'd prefer to see this coming from the `package.json` like so `import { name as LOG_NAME } from '../package.json'`. 

If you stick with a const, let's rename it to `PACKAGE_NAME`.